src/shared/translations/en.json

```diff
@@ -333,8 +333,9 @@
 							"progress-label": "withdrawal in progress..."
 						},
 						"disabled": {
-							"tooltip": "Market Creator has disabled all. withdrawals - sUSD bids are final"
-						}
+							"tooltip": "Market Creator has disabled all. withdrawals - sUSD bids are final",
```

Comment:
Looks like there is an extra period here

Reply:
yeah looks funny on the diff, but its because there is a new key right after it.

Reply:
are you sure? `Market Creator has disabled all. withdrawals` seems wrong, but if you are confident it's ok.

Reply:
ah I see what you mean! yeah let me fix that.